src/utl/include/utl/Logger.h

```diff
@@ -95,6 +95,8 @@ enum ToolId
  SIZE // the number of tools, do not put anything after this
 };
 
+#ifndef SWIG
```

Comment:
These are unnecessary, just use
#define __attribute__(x)
in Logger-py.i and change 
```
-            typename = std::enable_if_t<std::is_arithmetic<T>::value>>
+            typename U = std::enable_if_t<std::is_arithmetic<T>::value>>
```
to make swig happy